README.md

```diff
@@ -69,7 +69,7 @@ Target Account Actions:
 This Account Actions:
  1. Add your stack name to the `stacks` section of `riff-raff.yaml`
  1. Use riff-raff to upload the artifact to the dist bucket in your account (use preview and select just the appropriate upload tasks).  
- 1. Manually deploy the cloudformation template (at `template.yaml`) for the first time, filling in parameters as desired (retention days etc - at the Guardian I recommend that you set `OptionLowerFirstCharOfTags` to true). The stack name in the cfn deployment must match the value above
+ 1. Manually deploy the cloudformation template (at `template.yaml`) for the first time, filling in parameters as desired (retention days etc - at the Guardian I recommend that you set `OptionLowerFirstCharOfTags` to true). The CFN stack must be tagged with Stage = `CODE`, App = `cloudwatch-logs-management` and Stack = `<stack-name-from-step-1>` (e.g. Stack = `frontend` if you were deploying it to the frontend account).
```

Comment:
Could we script this?

Reply:
Yeah. I mean if we also put the retention days in as a parameter.

Actually it's now set as a default, which I suspect means that it has been overriden on all accounts.

Reply:
Hmmm, yes - every account now has a retention of 7 for everything.

Reply:
Think this is a little bug in Riff-Raff if I'm honest.